cupy/core/core.pyx

```diff
@@ -157,7 +157,7 @@ cdef class ndarray:
             'shape': self.shape,
             'typestr': self.dtype.str,
             'descr': self.dtype.descr,
-            'version': 2,
+            'version': 3,
```

Comment:
When `CUDA_ARRAY_INTERFACE_EXPORT_STREAM` is disabled, the `version` should be 2?